packages/mobile/src/navigator/types.tsx

```diff
@@ -112,9 +112,14 @@ export type StackParamList = {
   [Screens.RegulatoryTerms]: undefined
   [Screens.Security]: undefined
   [Screens.SelectLocalCurrency]: undefined
-  [Screens.Send]: undefined
+  [Screens.Send]:
+    | {
```

Comment:
what does the pipe before the type do? between them its like an OR but before?

Reply:
Yeah, Prettier adds it automatically. It is convenient for reading. 
```
...:
  | a
  | b
```
instead of
```
...: a 
  | b
```